### PR TITLE
chore: YoStarJP ocr fix

### PR DESCRIPTION
**Pull request by Manicsteiner**

### CODE REVIEW

File: resource/global/YoStarJP/resource/tasks.json [hunk 1]
```diff
@@ -2812,6 +2824,14 @@
...
                 "アーモンド",
                 "杏仁"
+            ],
+            [
+                "Friston-$",
+                "Friston-3"
+            ],
+            [
+                "U-[oO]ficial",
+                "U-Official"
```
Comment (ABA2396) [inline resource/global/YoStarJP/resource/tasks.json:2834]: image
参考这个吧

**Comment by Constrat** [comment]
In EN fortunately there no issues regarding this but.
For KR and JP isn't the CN official regex enough?

File: resource/global/YoStarJP/resource/tasks.json [hunk 2]
```diff
@@ -2812,6 +2824,14 @@
...
                 "アーモンド",
                 "杏仁"
+            ],
+            [
+                "Friston-$",
+                "Friston-3"
+            ],
+            [
+                "[Uu]-[O0o]f{1,2}icial",
+                "U-Official"
```
Comment (Constrat) [inline resource/global/YoStarJP/resource/tasks.json:2834]:
```suggestion
```

Comment (Constrat) [reply resource/global/YoStarJP/resource/tasks.json:2834]: Shouldn't this work as there is already the CharsReplaceOcr in `tasks.json`

**Comment by Constrat** [comment]
Just repeated the commit changing the name to allow merging.
(I think I finally understood how to fix the name thing through reset -> force push

**Pull request merged**